libs/stats/odc/stats/_gm_ls_bitmask.py

```diff
@@ -162,15 +164,17 @@ def reduce(self, xx: xr.Dataset) -> xr.Dataset:
         gm = geomedian_with_mads(xx, **cfg)
         gm = gm.rename(self.renames)
 
-        # Rescaling gm bands between 0-10000 (approx) range
-        # for band in gm.data_vars:
-        #     if band in self.bands:
-        #         gm[band] = scale * sr_scale * gm[band] + offset * sr_scale
-        #         gm[band] = gm[band].round().astype(np.uint16)
-        #
-        # # Rescale edev to sr_scale
-        # gm['emad'] = scale * sr_scale * gm['emad']
-        # gm['emad'] = gm['emad'].round().astype(np.uint16)
+        # rescale gm bands into surface reflectance scale
+        for band in gm.data_vars.keys():
+            if band in self.bands:
+                # gm[band] with low value (ie gm[band] < 7270) end up in negative values during sr scaling
+                # so resetting nodata and negative values back to 0 before rounding to uint16
```

Comment:
@fangfy @cbur24 see the comment above and let me know if we need to adjust this calculation

Reply:
Matches with what we discussed on Friday. Happy to see this tested.